### ISSUE TITLE
[BUG] in SentimentAnalyzer

### ISSUE THREAD

**Issue by SergiSvK** [pysentimiento/pysentimiento#57]
**Describe the bug**
When trying to create a sentiment analyzer with pysentimiento in Colab, I get an ImportError that says I need to install the accelerate package with a version equal or higher than 0.20.1 to use the PyTorch Trainer. However, I have already installed the accelerate package with version 0.21.0 and the error persists.

**To Reproduce**

Please provide a working code of your error and the stack trace or output related to it.

It could be a Colab notebook or just a snippet of code

```python
from pysentimiento import SentimentAnalyzer

analyzer = SentimentAnalyzer(lang="es")
```

**Expected behavior**

I expected to be able to create and use the sentiment analyzer without problems.

**Environment**

Please copypaste the output of `pip freeze` and `python --version` to help us understand the environment in which the bug occurred

```
accelerate==0.21.0
pysentimiento==0.7.1
torch==2.0.1+cu118
transformers==4.31.0
Python 3.10.12
```

**Additional context**

I am using Google Colab with a GPU available.

**Error message**

```
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-23-928f560478a6> in <cell line: 6>()
      4 transformers.logging.set_verbosity(transformers.logging.ERROR)
      5 
----> 6 analyzer = create_analyzer(task="sentiment", lang="es")

7 frames
/usr/local/lib/python3.10/dist-packages/transformers/training_args.py in _setup_devices(self)
   1714         if not is_sagemaker_mp_enabled():
   1715             if not is_accelerate_available(min_version="0.20.1"):
-> 1716                 raise ImportError(
   1717                     "Using the `Trainer` with `PyTorch` requires `accelerate>=0.20.1`: Please run `pip install transformers[torch]` or `pip install accelerate -U`"
   1718                 )

ImportError: Using the `Trainer` with `PyTorch` requires `accelerate>=0.20.1`: Please run `pip install transformers[torch]` or `pip install accelerate -U`
```

**Comment by finiteautomata** [comment]
@SergiSvK thanks for pointing out this issue! I will try to solve this ASAP. In the meanwhile, just `pip install accelerate` to fix it.

**Comment by jorgeston** [comment]
I had the same issue, then I installed  accelerate via pip and then I restarted the kernel and it worked

**Comment by finiteautomata** [comment]
Fixed in 0.7.2
